quickwit-storage/src/object_storage/s3_compatible_storage.rs

```diff
@@ -454,6 +454,17 @@ async fn download_all(byte_stream: &mut ByteStream, output: &mut Vec<u8>) -> io:
 
 #[async_trait]
 impl Storage for S3CompatibleObjectStorage {
+    async fn check(&self) -> anyhow::Result<()> {
+        self.s3_client
+            .list_objects(ListObjectsRequest {
```

Comment:
Sorry, I should have been more precise and asked you to use `list_objects_v2`.

Reply:
No problem! I updated the code.